postgres/postgres_test.go

```diff
@@ -51,11 +53,25 @@ func generateTrips(opts ...optTrips) []karma.Trip {
 }
 
 func TestStoreTrips(t *testing.T) {
-	err := postgres.StoreTrips(context.TODO(), conn, generateTrips())
+	trips := generateTrips(1000)
+	err := postgres.StoreTrips(context.TODO(), conn, trips)
 	require.NoError(t, err)
 
-	f, err := os.Create("trips.csv")
+	var buf bytes.Buffer
+	_, err = conn.PgConn().CopyTo(context.TODO(), &buf, "COPY (SELECT json_agg(row_to_json(trips)) FROM trips) TO stdout")
 	require.NoError(t, err)
-	_, err = conn.PgConn().CopyTo(context.TODO(), f, "COPY trips TO stdout")
+
+	gotTrips := make([]karma.Trip, 0, len(trips))
```

Comment:
same as above.. it's already obvious so better to make it concise
```suggestion
	got := make([]karma.Trip, 0, len(trips))
```
or
```suggestion
	res := make([]karma.Trip, 0, len(trips))
```
where res is a common var name for results (of a query/network call etc)